src/cli_shared/snapshot.rs

```diff
@@ -72,11 +76,31 @@ pub async fn fetch(
         .date_and_height_and_forest();
     let filename = filename(vendor, chain, date, height, forest_format);
 
-    match download_aria2c(&url, directory, &filename).await {
+    download_file_with_retry(url, directory, &filename).await
+}
+
+pub async fn download_file_with_retry(
+    url: Url,
+    directory: &Path,
+    filename: &str,
+) -> anyhow::Result<PathBuf> {
+    Ok(retry(
+        RetryArgs {
+            timeout: None,
+            max_retries: Some(3),
+            delay: Some(Duration::from_secs(60)),
```

Comment:
Do we want to have it somehow parametrized?

Reply:
I feel it's better to not ask the function caller to decide these parameters, how about changing it to 
```rust
RetryArgs {
    timeout: None,
    ..Default::default()
}
```

Reply:
Sounds good